src/gui/plugins/copy_paste/CopyPaste.qml

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+
+Rectangle {
+  width: 300
+  height: 200
+  color: "blue"
```

Comment:
Are you planning to add copy / paste buttons like we have on Gazebo classic?

![image](https://user-images.githubusercontent.com/5751272/132461128-1605f1b7-8f50-4f94-b4ac-650f15e3acae.png)

Reply:
Yeah, the blue box was just a placeholder while I worked on getting the right click functionality to work. I tried adding the buttons in https://github.com/ignitionrobotics/ign-gazebo/pull/1013/commits/cc3db6b0a6f75d29078a012a156f6e81f4d526f2, but the icons don't appear (the buttons still work - you can see what is copy and what is paste by hovering over the plugin menu). Any idea why the icons aren't visible?

![copy_paste_no_icon_buttons](https://user-images.githubusercontent.com/42042756/132772228-b4730ef3-fb50-4d1d-912c-dee80cda99a1.gif)

Reply:
Do you see any warnings / errors?

Reply:
The missing icons should be fixed with https://github.com/ignitionrobotics/ign-gazebo/pull/1013/commits/0b5942c37b10115b03ecce760a38b342336a9d90. Should I also follow the [Light Spawning PR](https://github.com/ignitionrobotics/ign-gazebo/pull/587) and add the copy/paste buttons to the [default GUI config](https://github.com/ignitionrobotics/ign-gazebo/blob/main/src/gui/gui.config)?